feature-proxy-manager/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
Maybe delte this and we'll add it if we actually need some rules?

Reply:
👍